.travis.yml

```diff
@@ -79,3 +88,12 @@ jobs:
       script:
         - bundle exec jekyll build
         - bundle exec htmlproofer ./_site --only-4xx --check-favicon --check-html --assume-extension
+
+env:
+  global:
+  # to create secure for travis-ci.com
```

Comment:
I don't really understand the comments.
Why repository should be cloned? (ours or a specific one from travis.org)
Where do we find the application `travis`?

Reply:
Just the same comments as for scripts. Our repository must be cloned. https://github.com/travis-ci/travis.rb